CHANGELOG.md

```diff
@@ -1,3 +1,91 @@
+# v1.11.0 (2022-11-15)
+
+## OS Changes
+
+* Prevent a panic `early-boot-config` when there is no IMDS Region ([#2493])
+* Update grub to 2.06-42 ([#2503])
+* Allow interface configuration via Ethernet permanent address ([#2519])
+* Allow bootstrap containers to manage swap ([#2537])
+* Add systemd analyze commands to troubleshooting log collection tool ([#2550])
+* Allow bootstrap containers to manage network configuration ([#2558])
+* Serialize bootconfig values correctly when the value is empty ([#2565])
+* Update zlib, libexpat, libdbus, docker-cli ([#2583])
+* Update host containers ([#2574])
+* Unmask /sys/firmware from host containers in host-ctr ([#2573])
+
+## Orchestrator Changes
+
+### ECS
+
+* Add additional ECS API configurations ([#2527])
```

Comment:
It would be nice to say what was added - maybe multiple bullet points mapping to the same PR.